src/main/java/net/dries007/tfc/objects/blocks/BlocksTFC.java

```diff
@@ -297,6 +431,37 @@ public static void registerBlocks(RegistryEvent.Register<Block> event)
         normalItemBlocks.add(new ItemBlockTFC(register(r, "sea_ice", new BlockIceTFC(FluidsTFC.SALT_WATER.get()), CT_MISC)));
 
         normalItemBlocks.add(new ItemBlockLargeVessel(register(r, "ceramics/fired/large_vessel", new BlockLargeVessel(), CT_POTTERY)));
+        {
+            Builder<ItemBlock> raw = ImmutableList.builder();
+            Builder<ItemBlock> polished = ImmutableList.builder();
+            Builder<ItemBlock> bricks = ImmutableList.builder();
+
+            raw.add(new ItemBlockTFC(register(r, "alabaster/raw/plain", new BlockDecorativeStone(MapColor.SNOW), CT_DECORATIONS)));
+            for (EnumDyeColor dyeColor : EnumDyeColor.values())
+            {
+                raw.add(new ItemBlockTFC(register(r, "alabaster/raw/" + dyeColor.getDyeColorName(), new BlockDecorativeStone(MapColor.getBlockColor(dyeColor)), CT_DECORATIONS)));
```

Comment:
`EnumDyeColor#getDyeColorName` is annotated as `SideOnly(Side.CLIENT)`. Use `getName` instead, as it returns the same field.